wrappers/ios/ci/build.sh

```diff
@@ -227,79 +225,44 @@ copy_lib_tocombine() {
-                    strip -S -x -o ${BUILD_CACHE}/arch_libs/${library}-$arch-stripped.a -r ${BUILD_CACHE}/arch_libs/${library}_${arch}.a
-                elif [ ! -f ${BUILD_CACHE}/arch_libs/${library}-$arch-stripped.a ]; then
-                    strip -S -x -o ${BUILD_CACHE}/arch_libs/${library}-$arch-stripped.a -r ${BUILD_CACHE}/arch_libs/${library}_${arch}.a
-                fi
-                source_libraries="${source_libraries} ${BUILD_CACHE}/arch_libs/${library}-$arch-stripped.a"
-            else
-                source_libraries="${source_libraries} ${BUILD_CACHE}/arch_libs/${library}_${arch}.a"
-            fi
+          libs_to_combine_paths="${libs_to_combine_paths} ${BUILD_CACHE}/arch_libs/${library}_${arch}.a"
         done
 
-        echo "Using source_libraries: ${source_libraries} to create ${BUILD_CACHE}/arch_libs/${COMBINED_LIB}_${arch}.a"
-        rm -rf "${BUILD_CACHE}/arch_libs/${COMBINED_LIB}_${arch}.a"
-        $libtool -static ${source_libraries} -o "${BUILD_CACHE}/arch_libs/${COMBINED_LIB}_${arch}.a"
-        source_combined="${source_combined} ${BUILD_CACHE}/arch_libs/${COMBINED_LIB}_${arch}.a"
-
-        lipo -info ${BUILD_CACHE}/arch_libs/${COMBINED_LIB}_${arch}.a
-
-        # TEMPORARY HACK (build libvcx without duplicate .o object files):
-        # There are duplicate .o object files inside the libvcx.a file and these
-        # lines of logic remove those duplicate .o object files
-        rm -rf ${BUILD_CACHE}/arch_libs/tmpobjs
-        mkdir ${BUILD_CACHE}/arch_libs/tmpobjs
-        pushd ${BUILD_CACHE}/arch_libs/tmpobjs
-        ar -x ../${COMBINED_LIB}_${arch}.a
-        ls >../objfiles
-        xargs ar cr ../${COMBINED_LIB}_${arch}.a.new <../objfiles
-        if [ "$DEBUG_SYMBOLS" = "nodebug" ]; then
-            strip -S -x -o ../${COMBINED_LIB}_${arch}.a.stripped -r ../${COMBINED_LIB}_${arch}.a.new
-            mv ../${COMBINED_LIB}_${arch}.a.stripped ../${COMBINED_LIB}_${arch}.a
-        else
-            mv ../${COMBINED_LIB}_${arch}.a.new ../${COMBINED_LIB}_${arch}.a
-        fi
-        popd
+        COMBINED_LIB_PATH=${BUILD_CACHE}/arch_libs/${COMBINED_LIB}_${arch}.a
+        echo "Going to combine following libraries: '${libs_to_combine_paths}' to create combined library: '$COMBINED_LIB_PATH'"
+        rm -rf "$COMBINED_LIB_PATH"
+        libtool -static ${libs_to_combine_paths} -o "$COMBINED_LIB_PATH"
+        combined_libs_paths="${combined_libs_paths} $COMBINED_LIB_PATH"
     done
 
-    echo "Using source_combined: ${source_combined} to create ${COMBINED_LIB}.a"
-    # Merge the combined library for each architecture into a single fat binary
-    lipo -create $source_combined -o $OUTPUT_DIR/${COMBINED_LIB}.a
+    for arch in ${ARCHS[*]}; do
+        COMBINED_LIB_PATH=${BUILD_CACHE}/arch_libs/${COMBINED_LIB}_${arch}.a
+        echo "Lipo info about combined library ${COMBINED_LIB_PATH}:"
+        lipo -info "$COMBINED_LIB_PATH"
+    done
```

Comment:
I really like how much unnecessary code is removed, it makes this piece much more readable. However, I think it would be even more readable if we remove code useful primarily for debugging, like the usages of `lipo -info ...` :)